src/barkha/WebTableFuntions.java

```diff
@@ -0,0 +1,77 @@
+/*1) Find how many employees reporting to each manager ID. 
+2) Find maximum employees reporting to which manager ID.
+*/
+package barkha;
+
+import java.util.HashMap;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+import barkha_base.PredefinedFunctions;
+
+public class WebTableFuntions extends PredefinedFunctions {
+	
+	int empCount=0;
+	String managerID;
+	WebDriver driver;
+	
+	void setUp() {
+		driver=start();
+	}
+	
+	void handleWebTable() throws InterruptedException {
+		driver.findElement(By.linkText("Demo Tables")).click();
+		Thread.sleep(2000);
+		int totalEmpCount=driver.findElements(By.xpath("//*[@class='table table-striped']/tbody/tr/td[4]")).size();
+		
+		HashMap<String,Integer> hashMap=new HashMap<String, Integer>();
+		
+		for (int index=1; index<=totalEmpCount; index++) {
+			String empManagerID=driver.findElement(By.xpath("//*[@class='table table-striped']/tbody/tr["+index+"]/td[4]")).getText();
+			
+			if (hashMap.containsKey(empManagerID)) {
+				hashMap.put(empManagerID, hashMap.get(empManagerID)+1);
+				
+				if (hashMap.get(empManagerID)>empCount){
```

Comment:
very good job @BarkhaPJun20 
keep it up, adjusting in single loop, it means you checking time complexity n space complexity as well good good